src/MCPServer/lib/settings/common.py

```diff
@@ -74,6 +76,23 @@
 max_overflow = 40
 port = 3306
 engine = django_mysqlpool.backends.mysqlpool
+
+[email]
+backend = django.core.mail.backends.console.EmailBackend
+host = smtp.gmail.com
+host_password =
+host_user = your_email@example.com
+port = 587
+ssl_certfile =
+ssl_keyfile =
+use_ssl = False
+use_tls = True
+file_path =
+amazon_ses_region = us-east-1
+default_from_email = webmaster@example.com
+subject_prefix = [Archivematica]
+timeout = 300
+#server_email =
```

Comment:
While I brought the config mappings and settings extraction into a common module, I felt the defaults would be more app-specific so left those in the settings file for each component.